pkg/bisect/bisect.go

```diff
@@ -29,13 +34,15 @@ type Config struct {
 }
 
 type KernelConfig struct {
-	Repo      string
-	Branch    string
-	Commit    string
-	Cmdline   string
-	Sysctl    string
-	Config    []byte
-	Userspace string
+	Repo           string
+	Branch         string
+	Commit         string
+	Cmdline        string
+	Sysctl         string
+	Config         []byte
+	BaselineConfig []byte
```

Comment:
Please add a comment here, an excerpt from this part of the PR descriptions would be good. It all looks useful, and since you already wrote it no reason to not add :)
```
Second patch is adding kernel configuration bisection as part of normal commit bisection. First original head with original commit is tested as earlier. Then original head is tested with configured baseline configuration (kernel_baseline_config). If crash reproduces with this baseline configuration it's used in commit bisection. If the crash doesn't reproduce then configuration bisection is run. When triggering configuration option is found provided baseline configuration is modified according the bisection results. This new configuration (kernel_minimalistic_configuration) is tested once more with current head. If crash reproduces with the generated configuration commit bisection is performed using it.

We are expecting configuration bisection to improve current commit bisection by killing "noise" caused by drivers not related to crash. It is also providing us possibility to identify which crashes certain kernel configuration is prone to.
```

Reply:
done